### PR TITLE
Wizard recipe: libcups-v2.4.2

### PR DESCRIPTION
**Pull request by IanButterworth**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: libcups
* Version: v2.4.2

@staticfloat please review and merge.

- [ ] https://github.com/JuliaPackaging/Yggdrasil/pull/5005

### CODE REVIEW

**Comment by IanButterworth** [comment]
Build prerequisites from https://github.com/michaelrsweet/libcups/blob/master/INSTALL.md
```
sudo apt-get install autoconf build-essential libavahi-client-dev \
     libgnutls28-dev libnss-mdns zlib1g-dev
```

Looks like libnss-mdns is needed first

File: L/libcups/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcups"
+version = v"2.3.3"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/michaelrsweet/libcups.git", "1fbe09e7a69774b2be4e10e12f13f8f3bf751f59")
```
Comment (giordano) [inline L/libcups/build_tarballs.jl:10]: https://github.com/openprinting/cups looks a more "official" repo (and it's what's linked in https://openprinting.github.io/cups/)

Comment (IanButterworth) [reply L/libcups/build_tarballs.jl:10]:
```suggestion
    GitSource("https://github.com/openprinting/cups.git", "12ff481989924fc82ffdf1b8699753a486a33547")
```

File: L/libcups/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcups"
+version = v"2.3.3"
```
Comment (IanButterworth) [inline L/libcups/build_tarballs.jl:6]:
```suggestion
version = v"2.4.2"
```

File: L/libcups/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+version = v"2.4.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/openprinting/cups.git", "12ff481989924fc82ffdf1b8699753a486a33547")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/libcups.git
```
Comment (IanButterworth) [inline L/libcups/build_tarballs.jl:15]:
```suggestion
cd ${WORKSPACE}/srcdir/cups.git
```

File: L/libcups/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+version = v"2.4.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/openprinting/cups.git", "12ff481989924fc82ffdf1b8699753a486a33547")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/cups.git
```
Comment (giordano) [inline L/libcups/build_tarballs.jl:15]: I thought we fixed this bug already
```suggestion
cd ${WORKSPACE}/srcdir/cups
```

Comment (giordano) [reply L/libcups/build_tarballs.jl]: Looks like https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/1203 isn't in any release it

**Comment by imciner2** [comment]
Closing due to inactivity. Please submit a new PR on top of the current master branch if you still want to include this recipe.

**Pull request closed without merging**